pydra/engine/tests/test_shelltask_inputspec.py

```diff
@@ -0,0 +1,1137 @@
+                        "help_string": """
+            Rician/(Gaussian)
+            Employ a Rician or Gaussian noise model.
+            """,
+                        "allowed_values": ["Rician", "Gaussian"],
+                        "argstr": "-n",
+                    },
+                ),
+            ),
+            (
+                "shrink_factor",
+                attr.ib(
+                    type=int,
+                    default=1,
+                    metadata={
+                        "help_string": """
+                    (1)/2/3/...
+                    Running noise correction on large images can be time consuming.
+                    To lessen computation time, the input image can be resampled.
+                    The shrink factor, specified as a single integer, describes this
+                    resampling. Shrink factor = 1 is the default.
+                    """,
+                        "argstr": "-s",
+                    },
+                ),
+            ),
+            (
+                "patch_radius",
+                attr.ib(
+                    type=int,
+                    default=1,
+                    metadata={
+                        "help_string": "Patch radius. Default = 1x1x1",
+                        "argstr": "-p",
+                    },
+                ),
+            ),
+            (
+                "search_radius",
+                attr.ib(
+                    type=int,
+                    default=2,
+                    metadata={
+                        "help_string": "Search radius. Default = 2x2x2.",
+                        "argstr": "-r",
+                    },
+                ),
+            ),
+            (
+                "correctedImage",
+                attr.ib(
+                    type=str,
+                    metadata={
+                        "help_string": """
+                    The output consists of the noise corrected version of the input image.
+                    Optionally, one can also output the estimated noise image.
+                    """,
+                        "output_file_template": "{inputImageFilename}_out",
+                    },
+                ),
+            ),
+            (
+                "noiseImage",
+                attr.ib(
+                    type=str,
+                    metadata={
+                        "help_string": """
+                    The output consists of the noise corrected version of the input image.
+                    Optionally, one can also output the estimated noise image.
+                    """,
+                        "output_file_template": "{inputImageFilename}_noise",
+                    },
+                ),
+            ),
+            (
+                "output",
+                attr.ib(
+                    type=str,
+                    metadata={
+                        "help_string": "Combined output",
+                        "argstr": "-o [{correctedImage} {noiseImage}]",
```

Comment:
```suggestion
                        "argstr": "-o [{correctedImage}, {noiseImage}]",
```
just to be technically correct.